docs_CN/Endpoints.md

```diff
@@ -97,11 +97,12 @@ Sample responses 有下面的这些值:
 
 ```swift
 let requestClosure = { (endpoint: Endpoint<GitHub>, done: MoyaProvider.RequestResultClosure) in
-    var request = endpoint.urlRequest
-
-    // Modify the request however you like.
-
-    done(.success(request))
```

Comment:
@devSC However, our Chinese docs here are invalid. We should be using `try` and `do/catch`. Care to update these?

Reply:
yes, i will open a new PR.